README.md

```diff
@@ -57,6 +61,13 @@ Example:
 
 `Products.css` and CSS class `price-tag` will result in `products-price-tag`.
 
+### Types
```

Comment:
Dodałem konwencje nazywania typów z prefiksem 'T', aby można było używać takich samych jak nazwy widoków.

Reply:
To taka powszechna konwencja? Tzw notacja węgierska

Reply:
Jak robiłem kilka tutoriali, to się z tym spotkałem, więc obstawiam, że tak.
Sam nie chciałem tego używać, bo mi się po prostu nie podoba, ale od teraz będzie to mus, jeżeli chcemy zachować to samo nazewnictwo, ale jednocześnie rozróżniać elementy, bo problem jest przy importach tej samej nazwy.